docs/source/index.rst

```diff
@@ -9,15 +13,32 @@ On a single machine dask increases the scale of comfortable data from
 *fits-in-memory* to *fits-on-disk* by intelligently streaming data from disk
 and by leveraging all the cores of a modern CPU.
 
-Users interact with dask either by making graphs directly or through the *dask
-collections* which provide larger-than-memory counterparts to existing popular
-libraries:
+Dask use cases
+==============
+
+* **Scientific computing libraries:** Projects like `xray`_ and `scikit-image`_ 
+  use dask.array for fast interactive analysis of large numeric datasets.
+* **Large unstructured or semi-structured data:** Dask.bag excels at 
+  processing large JSON blobs and log files.
+* **Scaling pandas-like data analysis to large datasets:** Dask.dataframe 
+  implements a part of the pandas interface, and can process data files larger than 
+  available memory.
+* **Algorithm development:** Working directly with dask graphs or dask.imperative 
+  is an excellent way to implement and test new algorithms in fields such as linear 
+  algebra, optimization, and machine learning.
```

Comment:
I agree that use cases are good and that the current docs don't do a good job of giving a one-liner for what dask is.  I just don't think that where we started with this was the right approach.  I don't know what the right approach is.

Consider the dremel power tool; it's very powerful but not very well understood/known.  The one line description of a dremel is that "it's a hand held power tool that spins very fast and has a lot of connectors to do cool stuff."  This is as accurate as a one-line description can be but it isn't very informative about use cases.  This is analogous to my bit "dask is a low latency dynamic task scheduler"; it's accurate but not helpful to find use cases.

Alternatively we could say "Dremels sand, they bevel, they can help you to build shelves" and that might be helpful, but, taken out of context this leaves us thinking that this is all a dremel does.  If I'm not building a shelf then I should try something else.

What was up here before was a picture of things that had been made with a dremel.  It was a bit too far down the application spectrum, we had no idea why the dremel was useful nor the ability to understand if it would fit the problem that we care about.  Now, having pictures of things built with a dremel is very motivating, and I'm not against the idea, it just needs to be much more clearly spelled out that these are examples of existing use cases.  

I might be too critical here.  Perhaps there exists a few slight changes to the content above that would make me like this.  In general though I would encourage you to rethink this from the ground up rather than going from the original text.